src/components/AddAnimal.vue

```diff
@@ -1,39 +1,66 @@
 <template>
-  <v-row justify="center">
-    <v-dialog v-model="dialog" persistent max-width="600px">
-      <template v-slot:activator="{ on, attrs }">
-        <v-btn color="primary" dark v-bind="attrs" v-on="on">
-          Open Dialog
-        </v-btn>
-      </template>
-      <v-card>
-        <v-card-title>
-          <span class="text-h5">Add New Animal</span>
-        </v-card-title>
-        <v-card-text>
-          <v-container>
-            <input type="file" Choose File />
-            <br />
+  <div>
+    <v-row justify="center">
```

Comment:
Don't use row outside v-dialogue, it have default properties to be in center and responsive.